apps/flame-defi/app/components/mobile-nav/mobile-nav.tsx

```diff
@@ -80,12 +80,20 @@ function MobileNav() {
                 Pool
               </Link>
               {featureFlags.earnEnabled && (
-                <Link
-                  href="/earn"
-                  className={`font-medium text-grey-light text-base ${pathname === "/earn" ? "text-orange-soft" : ""}`}
-                >
-                  Earn
-                </Link>
+                <>
+                  <Link
+                    href="/earn"
+                    className={`font-medium text-grey-light text-base ${pathname === "/earn" ? "text-orange-soft" : ""}`}
```

Comment:
We should update these to be pathname.includes("/earn") or else the style won't be applied on nested routes